.github/workflows/_buildpacks-release.yml

```
@@ -75,10 +75,16 @@ jobs:
       - name: Rust cache
         uses: Swatinem/rust-cache@v2.6.2
 
+      # the version of `libcnb-cargo` installed here is kept in sync with the version of `libcnb-package`
+      # that the release automation CLI tooling depends on
       - name: Install libcnb-cargo
-        # TODO: Derive this version from the `libcnb-package` version in this repo's `Cargo.toml`,
-        # or at least add a CI check to ensure the two versions are in sync.
-        run: cargo install --locked libcnb-cargo@0.13.0
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          LOCKFILE="${{ runner.temp }}/Cargo-$RANDOM.lock"
+          gh api -H "Accept: application/vnd.github.v3.raw" "/repos/heroku/languages-github-actions/contents/Cargo.lock?ref=${{ inputs.languages_cli_branch }}" > $LOCKFILE
+          LIBCNB_PACKAGE_VERSION=$(yq -ptoml -oyaml '.package[] | select(.name == "libcnb-package") | .version' $LOCKFILE)
```

It seems this could be simplified:
- the file could be download from GitHub's raw URL saving needing to use the GitHub CLI along with custom headers/URL (something that people will be much less familiar with). The URL can be found by clicking the "raw" link in GitHub's UI, and for this file is eg: https://raw.githubusercontent.com/heroku/languages-github-actions/main/Cargo.lock
- the file contents can be piped straight to yq (since it supports reading from stdin: https://mikefarah.gitbook.io/yq/#quick-usage-guide) saving having to write to a temporary file

eg something like this:

```suggestion
          LOCKFILE_URL="https://raw.githubusercontent.com/heroku/languages-github-actions/${{ inputs.languages_cli_branch }}/Cargo.lock"
          LIBCNB_PACKAGE_VERSION=$( \
              curl --silent --show-error --fail --retry 5 --retry-all-errors --connect-timeout 10 --max-time 60 "${LOCKFILE_URL}" \
              | yq -ptoml -oyaml '.package[] | select(.name == "libcnb-package") | .version' \
          )
```